flow-server/src/test/java/com/vaadin/flow/server/frontend/NodeUpdaterTest.java

```diff
@@ -84,22 +84,28 @@ public void execute() {
     }
 
     @Test
-    public void getGeneratedModules_should_excludeByFileName()
+    public void getGeneratedModules_should_includeWebComponents()
```

Comment:
```suggestion
    public void getGeneratedModules_should_includeOnlyWebComponents()
```
Only webcomponents should be added an no others.